specs/olm.spec

```diff
@@ -14,6 +14,14 @@ Steps:
   * Subscribe to operator
   * Wait for TektonConfig CR availability  
   * Change enable-api-fields to "beta"
+  * Verify namespace "openshift-pipelines" exist
+  * Apply
+    | S.NO | resource_dir                   |
+    |------|--------------------------------|
+    | 1    | testdata/hub/tektonhub.yaml    |
+  * Switch to project "openshift-pipelines"
```

Comment:
`Validate RBAC` will fail after this step, maybe you could try to update step `"Configure GitHub token for git resolver in TektonConfig"` to use `store.targetNamespace` instead of `store.namespace`, then you would not need to switch the project